### PR TITLE
Update span status codes to match spec

### PR DESCRIPTION
**Pull request by drexler**
Reference issue: #234

### CODE REVIEW

File: opentelemetry-jaeger/src/lib.rs [hunk 1]
```diff
@@ -622,7 +622,7 @@ fn build_span_tags(
     }
 
     // Ensure error status is set
-    if status_code != StatusCode::OK && !user_overrides.error {
+    if status_code != StatusCode::Ok && !user_overrides.error {
```
Comment (jtescher) [inline opentelemetry-jaeger/src/lib.rs:625]: This should probably instead look for error now

**Review by jtescher — CHANGES_REQUESTED** [review]
Looks good @drexler, just one small request

File: opentelemetry-jaeger/src/lib.rs (hunk 1 above)
Comment (drexler) [reply opentelemetry-jaeger/src/lib.rs]: done.

**Review by jtescher — APPROVED** [review]
Looks good!

**Pull request merged**